src/avram/queryable.cr

```diff
@@ -243,6 +243,22 @@ module Avram::Queryable(T)
     end
   end
 
+  def group_count
+    database.query_all(
```

Comment:
We will want to make sure we raise an error here if you try to call it when there's no groups. Would be even better if we could make it a compile-time error  somehow...

Reply:
It still works if there aren't groups. It will return a count of all records with an empty array as the grouping key.

Reply:
Yep! It becomes 1 group of `[]`, which is canonically correct!